xcorim.cddl

```diff
@@ -1,65 +1,47 @@
-xcorim = 
-  #6.47113(unsigned-xcorim) / 
+corim-revocation =
+  #6.47113(unsigned-xcorim-map) /
   #6.47114(signed-xcorim)
 
 signed-xcorim = #6.18(COSE-Sign1-xcorim)
 
-cose-label = int / tstr
-cose-values = any
-
-protected-signed-xcorim-header = {
-  1 => int,                      ; algorithm identifier
-  3 => "application/xrim+cbor",  ; content type identifier
-  4 => bstr,                     ; key identifier
-  corim-meta-key => corim-meta   ; xcorim signature
-                                 ; attributes are the same
-                                 ; as defined by corim
-                                 ; schema
-  * cose-label => cose-values,
+protected-signed-xcorim-header-map = {
+  xcorim.alg-id => int
+  xcorim.content-type => "application/xrim+cbor"
+  xcorim.issuer-key-id => bstr
+  xcorim.meta => xcorim-meta-map
+  * cose-label => cose-values
 }
 
-corim-meta = {
-  ; xcorim-entity-map is pointing to the details regarding construction and signing of xCoRIM
-  ; compare tag-metadata tag-issuer these are different
-  ; things
-  xcorim.entity => one-or-more<xcorim-entity-map>
-  ; NOTE(henk) this needed a key to be valid, but I am not
-  ; sure what the semantics are
-  ? validity => validity-entry,
-  ; used for revocation - NOTE(tho) I'm not sure I
-  ; understand how is this supposed to work?
-  ? deny-id => $corim-id-type,
+xcorim-meta-map = {
+  xcorim.signer => xcorim-entity-map
+  xcorim.timestamp => time
 }
 
 xcorim-entity-map = {
-  xcorim.entity-name => text,
-  ? xcorim.reg-id => any-uri,
-  xcorim.role => one-or-more<$xrim-role>,
+  xcorim.entity-name => $entity-name-type-choice
```

Comment:
ISTM that all these would end up resembling each other *a lot*.  I don't see a real reason for using a specialised instead of a shared type. But I may be missing something.  Happy to be convinced otherwise!